tests/pkg-layering/main.yml

```diff
@@ -307,6 +307,93 @@
       vars:
         package: "{{ g_epel_pkg }}"
 
+- name: Package Layering - GPG Test
+  hosts: all
+  become: true
+
+  tags:
+    - gpg
+
+  tasks:
+    - name: Make /srv/localrepo directory
+      file:
+        path: /srv/localrepo
+        state: directory
+        mode: 0755
+
+    - name: Copy aht-dummy rpm to /srv/localrepo
+      copy:
+         src: '{{ playbook_dir}}/../../rpm/aht-dummy-1.0-1.noarch.rpm'
```

Comment:
LOL...that is just as ugly as before.  But I guess it is a bit more deterministic.

Reply:
Haha, yeah we still have to go back up two directories to get to the rpm directory.  I could symlink into the directory to make it cleaner.  WDYT?

Reply:
Let's leave it as is for now.  It would be nice if we could figure out a way to derive the root of the repo while running tests and set a fact.